axelor-account/src/main/java/com/axelor/apps/account/service/analytic/AnalyticAttrsServiceImpl.java

```diff
@@ -37,47 +51,66 @@ protected void addAttr(
   }
 
   @Override
-  public void addAnalyticAxisAttrs(Move move, Map<String, Map<String, Object>> attrsMap)
+  public void addAnalyticAxisAttrs(Company company, Map<String, Map<String, Object>> attrsMap)
```

Comment:
Can you check if you can factorize this method with the one in `MoveAttrsService` ?